fulp_modules/departmental_ert/code/hardsuits.dm

```diff
@@ -117,3 +117,30 @@
 	icon_state = "clownert_commander"
 	inhand_icon_state = "ert_clown"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/clown/commander
+
+// safety moth
+
+/obj/item/clothing/suit/space/eva/safety_moth
+	name = "Safety Moth safety vest"
+	desc = "The standard issue safety vest of the Safety Moth. Functions like a hardsuit and offers superb protection against environmental hazards."
+	icon_state = "hazard"
+	inhand_icon_state = "hazard"
+	flags_inv = null
+	armor = list(MELEE = 65, BULLET = 50, LASER = 50, ENERGY = 60, BOMB = 50, BIO = 100, RAD = 100, FIRE = 80, ACID = 80)
```

Comment:
mmmm, gamer armor

Reply:
Yup, same stats as the engie ERT hardsuit